api/api.yaml

```diff
@@ -151,8 +158,10 @@ paths:
         - name: projectIdOrName
           in: path
           required: true
+          description: The unique identifier or name of the project. 
           schema:
             type: string
+          example: project-id-123
```

Comment:
```suggestion
          example: my-project
```

Reply:
@renukashiwakoti can you revise all the examples of projectIdOrName to `my-project`?